civihr_employee_portal/civihr_employee_portal.module

```diff
@@ -6553,3 +6553,53 @@ function _civihr_employee_portal_create_default_reports_configuration() {
       ->execute();
   }
 }
+
+/**
+ * Internal function for fetching custom markup from template and Angular Js.
+ *
+ * @param String
+ *    Name of the resource to load template and angular js.
+ * @param String
+ *    Name of the module from where scripts are to be loaded
+ * @param String
+ *    Name of the module from where html is to be loaded
+ *
+ * @return String
+ *    Custom markup from template file.
+ */
+function civihr_get_markup_for_extension($resource = '', $scriptModuleName, $markupModuleName) {
```

Comment:
In Drupal, functions are usually prefixed with the module name. So, in this case, the prefix should be `civihr_employee_portal` instead of just `civihr`